src/app/create/[id]/createBlueprintSteps/ExtractFields.tsx

```diff
@@ -305,7 +305,9 @@ const ExtractFields = ({
     if (
       !regexGeneratedOutputs.length ||
       regexGeneratedOutputs.some((output) =>
-        Array.isArray(output) ? output.join('').includes('Error') : output.includes('Error')
+        Array.isArray(output)
+          ? output.join('').includes('Error')
+          : output && output?.includes('Error')
```

Comment:
```suggestion
          : typeof output === 'string' && output.includes('Error')
```
Using `typeof output === 'string'` ensures we only call `.includes()` on actual strings, preventing errors if output is null, undefined, or some other non-string type